mobile_app_connector/models/firebase_registration.py

```diff
@@ -125,8 +125,14 @@ def mobile_register(self, json_data, **params):
         firebase_id = params["firebaseId"]
         operation = params["operation"]
         partner_id = params.get("supId", None)
+        language = params.get("language", None)
```

Comment:
Better yet, for one line assignation:
```suggestion
        language = params.get("language") or None
```
You can do the same change for `partner_id` field.

Reply:
Thanks for the tip. I'm curious, how is it better actually ? more efficient computing vise ?

Reply:
It's not more efficient but it saves you some lines after (see my comment below for lines 132-135 that you can remove completely.